packages/picasso/src/AlertInline/AlertInline.tsx

```diff
@@ -56,13 +89,14 @@ export const AlertInline = forwardRef<HTMLDivElement, Props>(function Alert(
         alignItems='center'
         className={classes.iconWrapper}
       >
-        {renderAlertIcon(variant)}
+        {icon}
       </Container>
       <Typography
-        size='medium'
+        size={alertVariant && typographyProps[alertVariant].size}
```

Comment:
```suggestion
        size={typographyProps[alertVariant!].size}
```

Since we have `alertVariant` in `defaultProps`, we know that they will always be around. We will need to disable the eslint rule locally for this, but we do it a lot in the codebase.

Reply:
and we can even use `const { size, weight, color } = typographyProps[AlertVariant!]` so we have it only in one place